### PR TITLE
Add CreateCardForIssue

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
Migration of https://github.com/SonarSource/sonar-dotnet/blob/9071b131df2277c2a0d2b36ff1a2a97be5682ba0/.github/workflows/MilestoneIssue.yml

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
     "module": "commonjs",
     "target": "es6",
     "lib": ["es6"],
-    "sourceMap": true
+    "sourceMap": true,
+    "newLine":  "lf"
```
Comment (pavel-mikula-sonarsource) [inline tsconfig.json:7]: When rebuilding the project locally, CRLF is used. That's a bit painful to maintain as rebuild file is marked as changed, but the only change is LF->CRLF that is not pushed.

See: https://github.com/SonarSource/gh-action-lt-backlog/blob/master/.gitattributes
Docs: https://www.typescriptlang.org/tsconfig#newLine

File: CreateCardForIssue/ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# CreateCardForIssue
+
+Crate card for issue in selected column. Typically used to add issue on Kanban when it's milestoned.
```
Comment (costin-zaharia-sonarsource) [inline CreateCardForIssue/ReadMe.md:3]: typo
```suggestion
Create a card for the issue in the configured column. Typically used to add an issue on Kanban when it's added to a milestone.
```

File: CreateCardForIssue/action.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+name: 'Crate card for issue'
```
Comment (costin-zaharia-sonarsource) [inline CreateCardForIssue/action.yml:1]:
```suggestion
name: 'Create card for issue'
```

File: CreateCardForIssue/action.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+description: 'Crate card for issue in selected column'
```
Comment (costin-zaharia-sonarsource) [inline CreateCardForIssue/action.yml:2]:
```suggestion
description: 'Create a card for issue in the configured column'
```

**Review by costin-zaharia-sonarsource — COMMENTED** [review]
Some typos

File: CreateCardForIssue/ReadMe.md (hunk 2 above)
Comment (pavel-mikula-sonarsource) [reply CreateCardForIssue/ReadMe.md:3]: I kept the `milestoned`, as that's how GitHub calls it, see
```
on:
  issues:
    types: ["milestoned"]
```

**Review by costin-zaharia-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**